### DISCUSSION TITLE
Copy file between client-server via clipboard

### DISCUSSION THREAD

**Discussion by x09** [neutrinolabs/xrdp#3266] [Q&A]
Client1 - Windows 10 (mstsc, printer redirection not checked)
Client2 - Linux, xfreerdp

Server - xrdp 0.10.1

Server -> Windows (client1)  copy/paste file from clipboard worked perfectly
Windows (client1) -> Server, not available, not working..
Linux (client2) -> Server,   not available, not working..

Why i can wrong?

grep -A 13 '^\[Channels' xrdp.ini | sed '/^;/d'

```
[Channels]
rdpdr=true
rdpsnd=true
drdynvc=true
cliprdr=true
rail=true
xrdpvr=true
```

grep ^allow_ch xrdp.ini
`allow_channels=true`

grep ^Restrict sesman.ini
```
RestrictOutboundClipboard=false
RestrictInboundClipboard=false
```

**Comment by matt335672** [comment]
Config looks fine.

Does plain text work?

**Reply by x09** [reply]
> Config looks fine.
> 
> Does plain text work?

plain text work

**Comment by matt335672** [comment]
Let's concentrate on the xfreerdp client for now.

- What desktop are you using (with version)?
- Can you write down the sequence of operations you are using to try to copy the file?

Thanks.
